spec/cohort.spec.js

```diff
@@ -0,0 +1,197 @@
+import Cohort from '../src/cohort.js'
+import { setId } from '../src/student.js'
+
+describe('Cohort', () => {
+  let cohort
+
+  beforeEach(() => {
+    cohort = new Cohort()
+    setId(1)
+  })
+
+  describe('when creating a new cohort', () => {
```

Comment:
I like that the describe descriptions flow into the individual test description.